src/cli/config.ts

```diff
@@ -185,6 +213,59 @@ export const NEXTJS_TS_CONFIG: AppConfig = {
   createInstallCommand: createNextCommand,
 };
 
+export const NUXTJS_CONFIG: AppConfig = {
+  templateId: "nuxtjs",
+  displayName: `Nuxt ${chalk.dim("(nuxi init)")}`,
+  language: "js",
+  templateDir: path.join(PKG_ROOT, "templates/nuxtjs"),
+  scaffoldingTool: "nuxi init",
+  twConfigExtension: ".js",
+    copyTemplate: async ({ projectDir }) => {
+    await fs.copy(
+      path.join(NUXTJS_CONFIG.templateDir, "app.vue"),
+      path.join(projectDir, "app.vue"),
+      );
+
+    await fs.copy(
+      path.join(NUXTJS_CONFIG.templateDir, "nuxt.config.js"),
+      path.join(projectDir, "nuxt.config.js"),
+    );
+  },
+  getCssOutputPath: ({ projectDir }) => {
+    return path.join(projectDir, "assets", "main.css");
+  },
+  createInstallCommand: createNuxtCommand,
+  deleteFiles: async ({ projectDir }) => {
+    await fs.remove(path.join(projectDir, "nuxt.config.ts"));
```

Comment:
Why do we need to remove nuxt.config.ts here?

Reply:
To allow the usage of JavaScript. Instead of nuxt.config.ts we will utilise nuxt.config.js